airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/streams.py

```diff
@@ -237,7 +237,7 @@ class Bans(IdIncrementalStream):
 
     def get_stream_data(self, response_data) -> List[dict]:
         bans = response_data["ip_address"] + response_data["visitor"]
-        bans = sorted(bans, key=lambda x: pendulum.parse(x["created_at"]))
+        bans = sorted(bans, key=lambda x: pendulum.parse(x["created_at"]) if x["created_at"] else pendulum.min)
```

Comment:
Can we leave `created_at` as `null` and don't change to `pendulum.min`?

Reply:
Sorry, just my mistake. Thought that we not only use `pendulum.min` for sorting, but also update the date.